src/Microsoft.AspNetCore.Rewrite/Internal/ApacheModRewrite/RuleBuilder.cs

```diff
@@ -176,12 +176,6 @@ public void AddRule(string rule)
                 _actions.Add(new ChangeCookieAction(flag));
             }
 
-            if (flags.GetValue(FlagType.Env, out flag))
-            {
-                // parse env
-                _actions.Add(new ChangeEnvironmentAction(flag));
```

Comment:
In most places we did throw either when parsing the flag or where it would have been implemented. Anything we can do to throw an error at startup telling them their rules contain unsupported features would be good.